polaris-react/src/components/Badge/README.md

```diff
@@ -260,7 +260,9 @@ Use to indicate when a given task has been partially completed. For example, whe
 Use to indicate when a given task has been completed. For example, when merchants have fulfilled an order.
 
 ```jsx
-<Badge progress="complete">Fulfilled</Badge>
+<Badge progress="complete" status="new">
+  Fulfilled
+</Badge>
```

Comment:
Btw, shouldn't bagdes which have `progress` status defined but no status have the same color by default?

Reply:
> Btw, shouldn't bagdes which with defined `progress` status but no status have the same color by default?

I think so, yes. If status isn't defined, the background colour should be surface neutral: https://github.com/Shopify/polaris/blob/866ae314cba596af78cef2e32bec929c20b65048/polaris-react/src/components/Badge/Badge.scss#L7